### ISSUE TITLE
Using react native maps directions with socket.io

### ISSUE THREAD

**Issue by vrinch** [bramus/react-native-maps-directions#91]
Hey guys, 
This is a great package and has been really helpful in the project I am currently embarking on, Thank you very much.

However, I am currently working an on demand app and I have been trying to figure out a way to draw a route from the current position of the servicer back to the location of the customer using socket.io, but I can’t seem to get it right. 
I have been able to emit the lng and lat of the client, but can’t draw a poly to the client location in the servicer app.
I know there’s a way to handle this using the google geocoder api and hard coding it.
I’d like to know if this can be achieved using this package.

I could post my code for better understanding at your request 
Many thanks

**Comment by bramus** [comment]
Please read the README. This package is created to “Draw a route between two [or more] coordinates, powered by the Google Maps Directions API”.

Which coordinates you pass into the component is up to you. They can be fixed, the user's location, a set of coordinates fetched from an API, etc. The routing is handled by the Google Directions API.

tl;dr yes
